CODEOWNERS

```diff
@@ -1 +1 @@
-* @aws-quickstart/ssp-eks-internal @aws-ia-ci @aws-quickstart/aws_quickstart_team
+* @aws-quickstart/ssp-eks-internal @aws-ia-ci @aws-quickstart/aws_quickstart_team @askulkarni2 @parkand1 @shapirov10
```

Comment:
@troy-ameigh is it possible to add me as another codeowner? 
Also Mikhail's alias is @shapirov103 .

Reply:
@youngjeong46 I added you in #567, hopefully that works.